README.rst

```diff
@@ -1,23 +1,87 @@
 Running Cyclus on Blue Waters
 =============================
 
-1. Login to a MOM node and lookup the docker image. Pull the image from
-   docker hub if not present.
+1. Login to a MOM node.
 
    .. code-block:: bash
 
         $ qsub -I -l walltime=00:30:00 -l nodes=1:ppn=1
+
+2. Load shifter and check if docker image exists on blue waters.
+
+   .. code-block:: bash
+
         $ module load shifter
-        $ getDockerImage lookup <REPO_NAME>
-        $ getDockerImage pull <REPO_NAME>
+        $ getDockerImage lookup arfc/cyclus_blue_waters:latest
+
+   If the image does not exist, pull from docker hub
+
+   .. code-block:: bash
+
+        $ getDockerImage pull arfc/cyclus_blue_waters:latest
 
-2. Run the `cyclus_blue_water.py` script with appropriate options (`--help` for help).
-   This will generate two files, cyclus_script.sh and pbs_script.pbs.
+3. Run the `cyclus_blue_water.py` script with appropriate options.
+   This will generate two files, `cyclus_script.sh` and `pbs_script.pbs`.
 
-3. Submit the job.
+   .. table::
+
+    +----------------+-------------------------------+
+    |``-h, --help``  |Show help message              |
+    +----------------+-------------------------------+
+    |``--nodes``     |Number of nodes                |
+    +----------------+-------------------------------+
+    |``--ppn``       |Processors per node            |
+    +----------------+-------------------------------+
+    |``--walltime``  |Wall time                      |
```

Comment:
Let's make sure that this is a clear definition of the variable. "Wall time" is jargon. How about "max time your job can run, hours:minutes:seconds"